app/views/subscription_mailer/notify_tag_added.html.erb

```diff
@@ -0,0 +1,13 @@
+<p>Dear <%= @user.username %>,</p>
```

Comment:
So here, this is great, but we don't need this yet -- not until the 5th checkbox down in your list: https://github.com/publiclab/plots2/issues/1421

So that could be its own PR, done after the `followers_who_dont_follow_tags` work, and we can publish `followers_who_dont_follow_tags` to the live site, well-tested, before we merge in templates like this. Each part by itself, independently -- how does that sound?

But be careful not to delete work from this branch -- you can make a backup before you break it up into separate PRs -- the work is excellent, i'm just asking for it to be done in smaller pieces. Thanks! 🎉